gst/nnstreamer/elements/gsttensor_trainer.c

```diff
@@ -493,6 +541,9 @@ gst_tensor_trainer_chain (GstPad * sinkpad, GstObject * parent,
   GstTensorMemory out_tensors[NNS_TENSOR_SIZE_LIMIT];
   GstTensorMetaInfo in_meta[NNS_TENSOR_SIZE_LIMIT];
   GstTensorMetaInfo out_meta[NNS_TENSOR_SIZE_LIMIT];
+  double data[4] =
+      { INVALID_VALUE, INVALID_VALUE, INVALID_VALUE, INVALID_VALUE };
```

Comment:
```suggestion
#include <math.h>

  double data[4] =
      { -INFINITY, -INFINITY, -INFINITY, -INFINITY };
```

Reply:
Thanks for the detailed explanation. I will use <math.h>